src/components/CountryLink.tsx

```diff
@@ -9,9 +9,18 @@ type TCountryLink = {
   className?: string;
   emptyContentFallback?: ReactNode;
   children?: ReactNode;
+  showFlag?: boolean;
+  showSlugAsChildren?: boolean;
 };
 
-export const CountryLink: FC<TCountryLink> = ({ countryCode, className = "", emptyContentFallback, children }) => {
+export const CountryLink: FC<TCountryLink> = ({
+  countryCode,
+  className = "",
+  emptyContentFallback,
+  children,
+  showFlag = true,
+  showSlugAsChildren = false,
```

Comment:
What instances do we need this?

Reply:
currently the country link renders an empty string if a children prop is not provided, as we want to actually show the slug value as well as create a link to the geo page we need to display that slug value aswell

Reply:
In that case can't we just pass in the slug as part of the children of the given link?

My worry with this is we are making the component be concerned with context